app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+import os
+
+
+def copy_file(command: str) -> None:
+    # Split the command into parts
+    parts = command.split()
+
+    # Check if the command has the correct format
+    if len(parts) != 3 or parts[0] != "cp":
+        print("Invalid command format. "
+              "Please use 'cp source_file target_file'.")
+        return
+
+    source_file = parts[1]
+    target_file = parts[2]
```

Comment:
```suggestion
    source_path, target_path = parts[1:]
```

Reply:
try do it like `_, source_path, target_path = parts`